cardinal/decorators.py

```diff
@@ -35,3 +35,20 @@ def inner(*args, **kwargs):
         return inner
 
     return wrap
+
+def event(triggers):
+    if isinstance(triggers, basestring):
+        triggers = [triggers]
+
+    if not isinstance(triggers, list):
+        raise TypeError("Event must be a trigger string or list of triggers")
+
+    def wrap(f):
+        @functools.wraps(f)
+        def inner(*args, **kwargs):
+            return f(*args, **kwargs)
+
+        inner.events = triggers
```

Comment:
I'm okay with the event naming here in the decorator because these are the events we trigger on. My event/callback comments really only apply to inside `PluginManager` when we're registering callbacks.